### PR TITLE
Mention capture of audio

### PR DESCRIPTION
**Pull request by henbos**
Fix for #61.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,8 +38,10 @@ <h2>
       </h2>
       <p>
         This document describes an extension to the Media Capture API [[GETUSERMEDIA]] that enables
-        the acquisition of a user's display, or part thereof, in the form of a video stream. This
-        enables a number of applications, including screen sharing using WebRTC [[WEBRTC]].
+        the acquisition of a user's display, or part thereof, in the form of a video stream. In some
+        cases system, application or window audio is also applicable which is presented in the form
```
Comment (alvestrand) [inline index.html]: applicable -> captured

File: index.html [hunk 2]
```diff
@@ -49,9 +51,9 @@ <h2>
         sandbox.
       </p>
       <p>
-        This document concerns itself primarily with the capture of video, but the general
-        mechanisms defined here could be extended to other types of media, of which audio
-        [[GETUSERMEDIA]] and depth [[MEDIACAPTURE-DEPTH]] are currently defined.
+        This document concerns itself primarily with the capture of video and audio [[GETUSERMEDIA]],
+        but the general mechanisms defined here could be extended to other types of media, of which
+        depth [[MEDIACAPTURE-DEPTH]] are currently defined.
```
Comment (alvestrand) [inline index.html]: are -> is

File: index.html [hunk 3]
```diff
@@ -38,8 +38,10 @@ <h2>
       </h2>
       <p>
         This document describes an extension to the Media Capture API [[GETUSERMEDIA]] that enables
-        the acquisition of a user's display, or part thereof, in the form of a video stream. This
-        enables a number of applications, including screen sharing using WebRTC [[WEBRTC]].
+        the acquisition of a user's display, or part thereof, in the form of a video stream. In some
+        cases system, application or window audio is also applicable which is presented in the form
+        of an audio stream. This enables a number of applications, including screen sharing using
```
Comment (alvestrand) [inline index.html]: audio stream -> audio track

File: index.html [hunk 4]
```diff
@@ -38,8 +38,10 @@ <h2>
       </h2>
       <p>
         This document describes an extension to the Media Capture API [[GETUSERMEDIA]] that enables
-        the acquisition of a user's display, or part thereof, in the form of a video stream. This
-        enables a number of applications, including screen sharing using WebRTC [[WEBRTC]].
+        the acquisition of a user's display, or part thereof, in the form of a video stream. In some
```
Comment (henbos) [inline index.html]: video track, audio track

**Comment by alvestrand** [comment]
@henbos will update to not allow the audio-only usecase.

File: index.html [hunk 5]
```diff
@@ -196,15 +198,27 @@ <h2>
...
+              audio sources to share. Which choices are available to choose from is up to the
+              user agent, and the audio source(s) are not necessarily the same as the video
+              source(s). An audio source may be a particular <a>application</a>, <a>window</a>,
+              <a>browser</a>, the entire system audio or any combination thereof. Unlike
+              <code><a>getUserMedia</a></code> with regards to audio+video, the user agent is
+              allowed not to return audio even if the audio constraint is present. If the user
+              agent knows no audio will be shared for the lifetime of the stream it MUST NOT
+              include an audio track in the resulting stream. The user agent MAY accept a
+              request for audio and video by only returning video, but MUST reject an
+              audio-only request if no audio track is available.
```
Comment (henbos) [inline index.html]: update to say audio only is not allowed

File: index.html (hunk 4 above)
Comment (henbos) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (henbos) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (henbos) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (henbos) [reply index.html]: Done

File: index.html (hunk 5 above)
Comment (henbos) [reply index.html]: Done

**Review by henbos — COMMENTED** [review]
Addressed all comments.

**Comment by henbos** [comment]
It's not clear from the PR what is needed to be done here before landing. But I think there was a request for updating the privacy section?

**Pull request merged**